Dockerfile.apt.template

```diff
@@ -18,7 +18,7 @@ RUN echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90tugboat && \
     \
     apt-get install -y \
         git locales sudo openssh-client ca-certificates tar gzip build-essential \
-        net-tools netcat-openbsd unzip zip bzip2 gnupg curl wget vim ssmtp runit \
+        net-tools netcat-openbsd unzip zip bzip2 gnupg curl wget vim runit \
```

Comment:
Is mail collection handled without ssmtp now?

Do we have release notes somewhere so if users had ssmtp in their scripts they know they need to install it now?

Reply:
https://github.com/Lullabot/tugboat/issues/3257